src/redux/actions/userActions.js

```diff
@@ -237,6 +237,34 @@ export const deleteUserFriends = (id, user) => (dispatch) => {
     });
 };
 
+
+export const getUserPromos = (id) => (dispatch) => {
+  dispatch({ type: types.GET_USER_FRIENDS_START, payload: true });
+  API.get(`savedPromos/users/${id}`)
+    .then((res) => {
+      console.log(res.data);
+      dispatch({
+        type: types.GET_USER_PROMOS,
+        payload: res.data,
+      });
+    })
+    .catch((err) => {
+      console.log(err);
+    });
+};
+
+export const addUserPromo = (user_id, promo_id) => (dispatch) => {
+  // dispatch({ type: types.GET_USER_FRIENDS_START, payload: true });
```

Comment:
do you think we can remove this unused code?

Reply:
yeah, thanks for catching both of those